### PR TITLE
[UI/buyer/checkout_complete] - create buyer/checkout_complete page 

### PR DESCRIPTION
**Pull request by Sayuriishida**
Additions:
- add function checkoutComplete() in BuyerController.php
- add buyers/checkout_complete.blade.php
- add route to buyers/checkout_complete.blade.php in web.php

### CODE REVIEW

File: resources/views/buyers/checkout.blade.php [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+
+    .card {
+        width: 40%;
+    }
+
+    .card-footer,
+    .card-header {
+        background-color: #f8fafc !important;
+    }
+</style>
```
Comment (Ditzler01) [inline resources/views/buyers/checkout.blade.php:23]: Please move this to separate css file.

File: resources/views/buyers/checkout_complete.blade.php [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        ul {
+            font-size: 16px;
+            line-height: 30px;
+            list-style: square;
+        }
+
+        ul li span {
+            vertical-align: 10px;
+        }
+    </style>
```
Comment (Ditzler01) [inline resources/views/buyers/checkout_complete.blade.php:21]: This one as well

File: resources/views/buyers/checkout_confirm.blade.php [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+
+    .card {
+        width: 40%;
+    }
+
+    .card-footer,
+    .card-header {
+        background-color: #f8fafc !important;
+    }
+</style>
```
Comment (Ditzler01) [inline resources/views/buyers/checkout_confirm.blade.php:23]: Same here

**Review by Ditzler01 — CHANGES_REQUESTED** [review]

File: resources/views/buyers/checkout.blade.php (hunk 1 above)
Comment (Sayuriishida) [reply resources/views/buyers/checkout.blade.php]: I put this in checkout.css with commit [72317eb](https://github.com/kredo-school/16th_rankshty/pull/45/commits/72317ebe156c5527809ac9d84121c000bbc9037d).

File: resources/views/buyers/checkout_complete.blade.php (hunk 2 above)
Comment (Sayuriishida) [reply resources/views/buyers/checkout_complete.blade.php]: I put this in checkout.css with commit [72317eb](https://github.com/kredo-school/16th_rankshty/pull/45/commits/72317ebe156c5527809ac9d84121c000bbc9037d).

File: resources/views/buyers/checkout_confirm.blade.php (hunk 3 above)
Comment (Sayuriishida) [reply resources/views/buyers/checkout_confirm.blade.php]: I put this in checkout.css with commit [72317eb](https://github.com/kredo-school/16th_rankshty/pull/45/commits/72317ebe156c5527809ac9d84121c000bbc9037d).

**Review by Ditzler01 — APPROVED** [review]

**Pull request merged**